apis/config/v1beta2/configuration_types.go

```diff
@@ -215,10 +215,6 @@ type ControllerConfigurationSpec struct {
 // WaitForPodsReady defines configuration for the Wait For Pods Ready feature,
 // which is used to ensure that all Pods are ready within the specified time.
 type WaitForPodsReady struct {
```

Comment:
If users want to enable w/ default parameters, how to specify that?
@mimowo @mbobrovskyi  Do you assume the following one?

```yaml
waitForPodsReady: {}
```

I am slightly histatin this way since It looks weird.

Reply:
Yeah, to specify default values you need to set:

```yaml
waitForPodsReady: {}
```

For me, it’s also not obvious. I think using the enable field is clearer. And I would propose to revert this and also https://github.com/kubernetes-sigs/kueue/pull/7583 PRs. 

But final decision for you @tenzen-y and @mimowo.